### ISSUE TITLE
Notify committer of failed builds via email

### ISSUE THREAD

**Issue by paulo-reichert** [GalateaRaj/fuse-starter-java#170]
Issue Description
--
There is no easy way to notify a committer via Slack that his/her commit broke the build. I suggest using emails for that.

Design
--
[Description of the root cause for the issue and how to resolve it. Should be complete before Review is approved]

Test Evidence
--
[include relevant tests and output. Should be complete before Review is approved]

Validation in Develop
-- 
[Proof issue is resolved in Develop. Should be complete before Issue is closed]

**Comment by gala-lightfoot** [comment]
It has become apparent Jenkins has issues mapping from email addresses to user accounts.  

https://stackoverflow.com/questions/45350740/jenkins-email-plugin-doesnt-send-email-to-user-who-broke-the-build

In our case, unless email prefixes match Git usernames, it will refuse to send the email (i.e. if your email is foo@bar.com, then your Git username must be foo).

A workaround is to enable _-Dhudson.tasks.MailSender.SEND_TO_UNKNOWN_USERS_ on Jenkins, but this is by no means an optimal solution due to the inherent security implications.

**Comment by GalateaRaj** [comment]
See note in slack

**Comment by GalateaRaj** [comment]
Per my slack post, I think we can put this one on hold for now.

**Comment by paulo-reichert** [comment]
I'll close this. If this becomes an issue worthy of someone's time we can re-open or create a new one.
